src/Sarif/CodeGenHints.json

```diff
@@ -479,16 +479,6 @@
       "kind": "DictionaryHint"
     }
   ],
-  "Result.HelpUri": [
```

Comment:
>HelpUri [](start = 10, length = 7)

Don't delete it -- this was a typo for the real property, `Rule.HelpUri`. Just change that and move it down into alphabetical order.

Reply:
Oh durr, you already did.

---
In reply to: [229077825](https://github.com/Microsoft/sarif-sdk/pull/1121#discussion_r229077825) [](ancestors = 229077825)